urbanstats-persistent-data/run_for_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd $(dirname $0)
+rm -r venv
+virtualenv -p python3 venv
+source venv/bin/activate
+pip3 install -r requirements.txt
```

Comment:
Maybe we only need to do this once rather than every test? Obviously we don't want to add a complex caching system, but maybe this is something we can do in the Dockerfile? (and maybe put the existing python libs in a venv)

This can be a separate MR (or not done at all until it starts really biting us)

Reply:
We can do that, but I'm pretty sure these end up being cached anyway, it just copies them over